src/report_workflow/test_report_runner.py

```diff
@@ -101,7 +101,11 @@ def component_entry(self, component_name: str, ci_group: int = None) -> Any:
 
         test_component = self.test_manifest.components[component_name]
 
-        config_names = [config for config in test_component.__to_dict__().get(self.test_type)["test-configs"]]
+        config_names = []
+        if "test-configs" in test_component.__to_dict__().get(self.test_type):
+            config_names = [config for config in test_component.__to_dict__().get(self.test_type)["test-configs"]]
+        elif "test-spec" in test_component.__to_dict__().get(self.test_type):
```

Comment:
Add some comments to be clear that `test-config` is for integ-test for `test-spec` is for smoke-test specifically.

Reply:
A better way would be to branch based on `self.test_type` instead of entries within `self.test_type`.

Reply:
Updated. It's now compare based on `self.test_type` content.